lib/llm-events/langchain/chat-completion-message.js

```diff
@@ -12,7 +12,6 @@ const { makeId } = require('../../util/hashes')
  * @typedef {object} LangChainCompletionMessageParams
  * @augments LangChainEventParams
  * @property {string} content The text of the response received from LangChain.
- * @property {string} role The role of the message, e.g. "human."
```

Comment:
`role` seems to still be defined as a `LlmChatCompletionMessage` property in the (wip) spec. Why have we removed it here?